phantom_token.c

```diff
@@ -224,27 +224,28 @@ static ngx_int_t handler(ngx_http_request_t *request)
     if (!module_location_config->enable)
     {
         ngx_log_debug0(NGX_LOG_DEBUG_HTTP, request->connection->log, 0, "Module disabled");
-
         return NGX_DECLINED;
     }
 
+    if (request->method == NGX_HTTP_OPTIONS)
```

Comment:
Single Page Applications should be able to use this module without other special NGINX directives